node/src/tasks/bsp_confirmed_storing.rs

```diff
@@ -0,0 +1,94 @@
+use sc_tracing::tracing::*;
```

Comment:
I'm gonna need some convincing for having this handler as a separate task. The way I see it, a `task` file would include the whole flow of what it's name describes. The `bsp_upload_file` should include the whole process of a BSP receiving a file from a user. If that includes more than one event handler, so be it, but the flow of logic is easier to follow than having to jump around files. So in short, I'd put this handler in `bsp_upload_file.rs`, but I'm open to discuss it.